frontend/src/Views/LandingPage.js

```diff
@@ -68,14 +116,40 @@ class LandingPage extends Component {
 
         if (this.state.office) {
             newStates = newStates.filter(
-                state => state.office_contact === true);
+                state => state.no_contact === true);
         }
 
         if (this.state.witness) {
             newStates = newStates.filter(
                 state => state.no_witness_required === true);
         }
 
+        if (this.state.population_filter === 1) {
+            newStates = newStates.filter(
+                state => state.population < 2500000);
```

Comment:
would recommend avoiding hard-coded values and just making a separate variable for 250000